dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListener.cpp

```diff
@@ -45,14 +50,26 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationListener::OnRawNotificationReceived
 }
 CATCH_RETURN()
 
-const std::string NotificationListener::ConvertProcessNameToUtf8String()
+const std::wstring NotificationListener::GetPayloadAsWideString(unsigned int payloadLength, byte* payload)
```

Comment:
To be consistent with the naming schema, this should be named as Utf8ToUtf16. The API should be global

Reply:
Also it might make sense to have all string conversion utility helpers in a seperate header definition file so that they can be resued everywhere in our code base.